### ISSUE TITLE
Old links to description of statistics for module preservation 

### ISSUE THREAD

**Issue by Dimmiso** [smorabit/scWGCNA#185]
Thanks a lot for supporting and developing hdWGCNA, really great package!
It seem that both links to description of statistics for module preservation (on page https://smorabit.github.io/hdWGCNA/articles/module_preservation.html) do not work (see attached screen shot). Can they be updated in the web-page mentioned above or provided any other way?  

Thanks a lot!

Dmitry
![Capture](https://github.com/smorabit/hdWGCNA/assets/45941025/7ffa990e-078a-4396-89b6-8fa92dafb7f3)

**Comment by smorabit** [comment]
Thank you for pointing out this issue. These were supposed to link to resources provided by the original WGCNA authors. Unfortunately it looks like their entire website is down at the moment. Clicking https://horvath.genetics.ucla.edu/ gives a website unavailable error.

I hope that this is just temporary and that it will be resolved soon but it is out of my control. If this problem persists I can include these descriptions in this tutorial.

**Comment by Dimmiso** [comment]
Hi, thanks for your answer! When I tested those and similar links to https://horvath.genetics.ucla.edu/ several weeks back they were at the same state, which is not very surprising since Steve Horvath moved from UCLA to Altos Lab, or? Taking into account this it would be logic that his materials are removed from UCLA webpage. It is not problem for me to check again in a week or two and remind if links still do not work. Thanks a lot for supporting hdWGCNA!

**Comment by smorabit** [comment]
Yeah I think that this could have to do with the move from UCLA to Altos since these links are from ucla.edu. I think if this doesn't get resolved in a few weeks, I will plan to simply remove these links and host the info directly within these tutorials. 

It would really be a shame if that website is not recovered because there were so many useful tutorials for WGCNA on there!

**Comment by DelongZHOU** [comment]
I checked the link today and it's not working.
I also tried web archive, unfortunately all the snapshots are from when the website is down.

Reading briefly from the paper 
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1001057

It seems higher the Z score and lower the rank indicate better preservation. Z score is affected by module size, while the rank is not. They proposed a cutoff for Z score (>10 preserved, <2 not), but I didn't find a hard cutoff for the rank.


Edit:

Sam, could you please explain the difference bewteen qual and pres? Thanks!

**Comment by smorabit** [comment]
Hi,

I am disappointed that the website is still down! I will provide detailed explanations of these stats in hdWGCNA as soon as possible, hopefully in the next few weeks.

> Sam, could you please explain the difference bewteen qual and pres? Thanks!

You can read more about this in the module preservation paper which you linked above but in general the authods group the stats into two main groups, where "qual" refers to stats regarding the quality of the network, while "pres" refers to stats directly regarding module preservation. The differences may seem nuanced so I encourage you to read the article. The module preservation stats are summarized in Table 1 of this paper. From the paper:

> Traditional cluster validation (or quality) statistics can be split into four broad categories: cross-tabulation, density, separability, and stability statistics.

**Comment by DelongZHOU** [comment]
Hi all,
I recently found Peter Langfelder is posting again on his website https://peterlangfelder.com/ and got his permission to share his Dropbox where he's put the old tutorials:
https://www.dropbox.com/scl/fo/4vqfiysan6rlurfo2pbnk/h?rlkey=3za224679kv8ulbpl5s9mwrkt&e=1&st=f1bi7e7p&dl=0
He also mentioned that he'll find a permanent place for these in the near future.
Best,
Delong
